mlflow/pyfunc/__init__.py

```diff
@@ -2092,11 +2092,11 @@ def spark_udf(
 
     if use_dbconnect_artifact:
         udf_sandbox_info = get_dbconnect_udf_sandbox_info(spark)
-        if Version(udf_sandbox_info.mlflow_version) < Version("2.19.0"):
+        if Version(udf_sandbox_info.mlflow_version) < Version("2.18.0"):
             raise MlflowException(
                 "Using 'mlflow.pyfunc.spark_udf' in Databricks Serverless or in remote "
                 "Databricks Connect requires UDF sandbox image installed with MLflow "
-                "of version >= 2.19."
+                "of version >= 2.18."
```

Comment:
The related [feature](https://github.com/mlflow/mlflow/pull/13276) is shipped in MLflow 2.18